external/stdk/tizenrt-wrapper/os/iot_os_util_tizenrt.c

```diff
@@ -281,14 +281,18 @@ static int check_for_timeout(clock_t *const ptime_out, clock_t *const pticks_to_
 
 void iot_os_timer_count_ms(iot_os_timer timer, unsigned int timeout_ms)
 {
-	((tizenrt_timer *)timer)->ticks_to_wait = MSEC2TICK(timeout_ms); /* convert milliseconds to ticks */
+	((tizenrt_timer *)timer)->ticks_to_wait = (timeout_ms == portMAX_DELAY) ? portMAX_DELAY : MSEC2TICK(timeout_ms); /* convert milliseconds to ticks */
```

Comment:
Should we handle `MSEC2TICK(wait_time_ms)` at line 122 in same way?
How about making a local macro `_MSEC2TICK` to handle portMAX_DELAY logic?

Reply:
although stdk have no requirement of this in the function at line 122, but for complete logic, it's better to refine this. thanks for suggestion.